app/code/Magento/Customer/Block/Address/Renderer/DefaultRenderer.php

```diff
@@ -172,9 +172,9 @@ public function renderArray($addressAttributes, $format = null)
             }
             $attributeCode = $attributeMetadata->getAttributeCode();
             if ($attributeCode == 'country_id' && isset($addressAttributes['country_id'])) {
-                $data['country'] = $this->_countryFactory->create()->loadByCode(
-                    $addressAttributes['country_id']
-                )->getName();
+                $data['country'] = $this->_countryFactory->create()
+                    ->loadByCode($addressAttributes['country_id'])
+                    ->getName(isset($addressAttributes['locale']) ? $addressAttributes['locale'] : null);
```

Comment:
Lets use the null coalescing operator
```suggestion
                    ->getName($addressAttributes['locale'] ?? null);
```